packages/abi/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@aragon/osx-abi",
+  "module": "index.ts",
+  "type": "module",
+  "version": "0.0.1",
+  "scripts": {
+    "build:contracts": "cd ../contracts && yarn build",
+    "generate:abi": "./scripts/generate-abi.sh",
+    "clean": "rimraf generated && rimraf index.ts",
+    "publish:npm": "./scripts/publish.sh"
+  },
+  "devDependencies": {
+    "@wagmi/cli": "^2.1.4",
```

Comment:
We could nohoist wagmi given that osx is a backend package and it will just bloat the root node modules folder anyway.